controllers/contacts.js

```diff
@@ -0,0 +1,86 @@
+const { Router } = require('express');
+const { Contact } = require('../models');
+const { dropUndefined } = require('../utils/dropUndefined');
+const { buildFilter } = require('../utils/queryHelper');
+const ObjectId = require('mongodb').ObjectID;
+
+const router = Router();
+
+// gets one contact by contact ID
+router.get('/:id', async (req, res) => {
+	const { id } = req.params;
+	let contactId = ObjectId(id);
+
+	try {
+		const contact = await Contact.findById(contactId);
```

Comment:
Entering a string representation of a document's ObjectId into mongoose's `findById` method automatically converts the string into an `ObjectId`.